app/src/main/java/in/avimarine/boatangels/fragments/BoatsForInspectionFragment.java

```diff
@@ -139,7 +138,7 @@ public BoatHolder onCreateViewHolder(ViewGroup group, int i) {
 
   }
   @Override
-  public View onCreateView(LayoutInflater inflater, ViewGroup container,
+  public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
```

Comment:
Make sure to catch cases of 'null inflater' that might be sent

Reply:
This denotes that the parameter can never be null. The parameter has no real meaning but if google places it there, they are promising that it will never be null